apps/web/src/app/_components/search/search-filter.tsx

```diff
@@ -16,11 +16,15 @@ const formSchema = z.object({
 
 export type SearchFilterFormType = typeof formSchema;
 
-export default function SearchFilter() {
+interface SearchFilterProps {
```

Comment:
i know this is so my inner fundies 1 ta but what are our thoughts on starting to us js doc to lightly document the functions. Espesh since dev turnover may be on the rise (oml we are getting old). Up to rishi hes TL i'm just throwing this out there ❤️ 🙈

Reply:
i agree with this to make it easier for everyone to read

Reply:
YES pls document with JSDoc 🙏🏻 I remember we were doing it for a bit at the start of Spring but that fizzled out lol.

Reply:
yessir